provider/cluster_rosa_classic_resource.go

```diff
@@ -693,18 +691,54 @@ func createClassicClusterObject(ctx context.Context,
 		builder.Version(vBuilder)
 	}
 
+	builder, err = buildProxy(state, builder)
```

Comment:
nit. It might be simpler if the function would return the proxy and not update the builder. We can then do `builder.Proxy(proxy)` here. This will make the code more readable, IMO. but it works :)

Reply:
I'm doing it because not all the properties are part of the "proxy", the AdditionalTrustBundle is property of the builder